pkg/api/dashboard.go

```diff
@@ -152,7 +152,15 @@ func (hs *HTTPServer) GetDashboard(c *models.ReqContext) response.Response {
 		hs.getAnnotationPermissionsByScope(c, &annotationPermissions.Organization, accesscontrol.ScopeAnnotationsTypeOrganization)
 	}
 
+	prefsQuery := pref.GetPreferenceWithDefaultsQuery{OrgID: c.OrgID, UserID: c.SignedInUser.UserID, Teams: c.Teams}
+	preference, err := hs.preferenceService.GetWithDefaults(c.Req.Context(), &prefsQuery)
+	if err != nil {
+		preference.HomeDashboardID = 0
```

Comment:
I don't think we need this given that we return immediately in the next line without referencing `preference` anywhere.